pandas/tests/frame/test_indexing.py

```diff
@@ -2626,6 +2626,17 @@ def test_index_namedtuple(self):
         result = df.loc[IndexType("foo", "bar")]["A"]
         assert result == 1
 
+    @pytest.mark.parametrize("tpl", [tuple([1]), tuple([1, 2])])
```

Comment:
this file moved to pandas/tests/frame/indexing/test_indexing.py

Reply:
thanks, @jreback